component/src/main/java/com/synopsys/integration/alert/component/settings/proxy/database/accessor/SettingsProxyConfigAccessor.java

```diff
@@ -49,14 +51,12 @@ public SettingsProxyConfigAccessor(
 
     @Override
     @Transactional(readOnly = true)
-    public long getConfigurationCount() {
-        return settingsProxyConfigurationRepository.count();
-    }
-
-    @Override
-    @Transactional(readOnly = true)
-    public Optional<SettingsProxyModel> getConfiguration(UUID id) {
-        return settingsProxyConfigurationRepository.findById(id).map(this::createConfigModel);
+    public Optional<SettingsProxyModel> getConfiguration() {
+        Page<SettingsProxyConfigurationEntity> proxyConfigPage = settingsProxyConfigurationRepository.findAll(pageRequest);
+        return proxyConfigPage.getContent()
```

Comment:
Do you want to log a warning if there are multiple proxy configs found? People can modify the database directly or we have run into issues with migration in the past that lead to multiple configs when there should only be one.

Reply:
It would make debugging a lot easier